src/extensions/block-editor-settings/index.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * WordPress Dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { CheckboxControl } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useEntityProp } from '@wordpress/core-data';
+import { useEffect } from '@wordpress/element';
+import { registerPlugin } from '@wordpress/plugins';
+
+/**
+ * Internal Dependencies
+ */
+import CoBlocksSettingsModalControl from '../coblocks-settings/coblocks-settings-slot';
+
+export const COLOR_PANEL_SETTINGS_KEY = 'coblocks_color_panel_controls_enabled';
+export const CUSTOM_COLORS_SETTINGS_KEY = 'coblocks_custom_colors_controls_enabled';
+export const GRADIENT_PRESETS_SETTINGS_KEY = 'coblocks_gradient_presets_enabled';
+
+function CoBlocksEditorSettingsControls() {
```

Comment:
@jrtashjian I wonder if the color related settings belong here at all? Maybe seperate then into their own concern?

Reply:
You're right, I didn't think enough on where to put this one. I can move it into `src/extensions/colors` instead.

Reply:
Yeah I feel it might be more appropriate